plugin.video.imdb.trailers/addon.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.imdb.trailers" name="IMDb Trailers" version="2.1.11" provider-name="gujal">
+<addon id="plugin.video.imdb.trailers" name="IMDb Trailers" version="2.1.11.1" provider-name="gujal">
```

Comment:
Could you use the `2.1.11+matrix.1` as version number? That will make it higher than the version in https://github.com/xbmc/repo-plugins/pull/2913. This is the preferred way at the moment.

Reply:
Changed as per advice.